content/pages/education/apply.md

```diff
@@ -95,3 +113,28 @@ You must apply for education benefits using eBenefits if you're:
 </div>
 
 <div markdown="0"><br></div>
+
+<script type="text/javascript">
+  // I'm open to suggestions on how to not do this here
```

Comment:
Create a separate js file and just import it here? That way it can also be used by other pages if this becomes an interaction we want to use elsewhere.